saskatoon/harvest/admin.py

```diff
@@ -69,25 +71,23 @@ class PropertyAdmin(LeafletGeoAdmin):
     model = Property
     inlines = [PropertyImageInline]
     list_display = (
-        '__str__',
-        'authorized',
+        'short_address',
+        'owner_edit',
+        'owner_type',
+        'owner_phone',
+        'owner_email',
         'pending',
-        'harvest_every_year', 
+        'authorized',
         'approximative_maturity_date',
-        'avg_nb_required_pickers',
         'neighborhood',
         'city',
         'postal_code',
+        'pk'
     )
     list_filter = (
+        PropertyOwnerTypeAdminFilter,
         'authorized',
         'pending',
-        'harvest_every_year',
-        'public_access',
-        'neighbor_access',
-        'compost_bin',
-        'ladder_available',
-        'ladder_available_for_outside_picks',
```

Comment:
What happened to those fields?

Reply:
I find having too many fields (especially with very long verbose_names) in the admin list_display makes the panel annoying to use due to the long horizontal scroll bar. Most of these fields can still be used as filters, I don't think they are primordial here personally.

Reply:
sorry my previous comment had to do with removing list_display fields, in this snippet I did remove filters so we don't have to scroll *vertically* for too long. I think such filters will be mostly used by core members/pickleaders through the `Properties` web view and are not super important for admins but i might be wrong (in the old generation there were no fields or filters at all)